src/index/Index.cpp

```diff
@@ -159,6 +160,11 @@ void Index::createFromNTriplesFile(const string& ntFile,
                    _patterns, _maxNumPatterns);
   }
   openFileHandles();
+  // we have deleted the vocabulary in the process to save ram, so now we have
```

Comment:
So since it is in `Index.cpp` it's not just for text indices?

Reply:
You're right, this is a duplicate.
The actual solution is by setting _onDiskLiterals and _onDiskBase before, then it suffices to read it  in Index.Text.cpp. I am going to fix this tomorrow.